calculation/calculation_rule.py

```diff
@@ -68,11 +71,34 @@ def check_calculation(cls, instance):
     @classmethod
     def calculate(cls, instance, *args):
         if instance.__class__.__name__ == "ContractContributionPlanDetails":
```

Comment:
I'm not sure it it's best approach for checking types. I'm aware that we cannot directly compare types as it'd require importing ContractContributionPlanDetails class, but can we use something like 
`if 'contribution_plan' in sys.modules and isinstance(instance, sys.modules['contribution_plan'].ContractContributionPlanDetails)` ?

Reply:
https://openimis.atlassian.net/wiki/spaces/OP/pages/2250473529/FS+calculation+rule+1+contribution+valuation - on wiki page - in "calculation" specification we have such condition defined by @delcroip 

```
if instance.__class__ == ContractContributionPlanDetails.
    rate = instance.contribution_plan.json_ext.rate
    income = instance.contract_details.json_ext.income
    CCPD.value = income  * rate
    return CCPD.value
else Return False 
```

That is why I used this approach. But I can define this as you wrote - check module and instance!

Reply:
Also I think that "ContractContributionPlanDetails" is related to the contract and we should check a Contract module instead of contribution_plan. 
`if 'contract' in sys.modules and isinstance(instance, sys.modules['contract'].ContractContributionPlanDetails)` ?